src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -1126,80 +1126,6 @@ public void TestReportingDiagnosticWithInvalidSpan()
                 );
         }
 
-        [Fact, WorkItem(13120, "https://github.com/dotnet/roslyn/issues/13120")]
-        public void TestRegisteringAsyncAnalyzerMethod()
-        {
-            string source = @"";
-            var analyzers = new DiagnosticAnalyzer[] { new AnalyzerWithAsyncMethodRegistration() };
```

Comment:
Is `AnalyzerWithAsyncMethodRegistration` used anywhere else? 

I think it could be deleted as well from `CommonDiagnosticAnalyzers`.

Reply:
Updated to remove `AnalyzerWithAsyncMethodRegistration` and `AnalyzerWithAsyncLambdaRegistration`.